src/main/scala/org/constellation/domain/redownload/RedownloadService.scala

```diff
@@ -343,7 +343,11 @@ class RedownloadService[F[_]: NonEmptyParallel](
             _ <- logger.debug(s"Alignment result: $result")
             _ <- logger.debug("Redownload needed! Applying the following redownload plan:")
             _ <- applyPlan(plan, meaningfulMajorityState).value.flatMap(F.fromEither)
+            _ <- metrics.updateMetricAsync("redownload_hasLastRedownloadFailed", 0)
           } yield ()
+        }.handleErrorWith { error =>
+          metrics.updateMetricAsync("redownload_hasLastRedownloadFailed", 1) >>
```

Comment:
what about incrementing that value instead of setting 0/1?

Reply:
We don't have the function for resetting the incremental metric. I didn't want to implement it at this moment. But that was also my thought initially to have it increment and reset when it's successful.

Reply:
nah, why we ever need to reset it? just increment in case of failure

Reply:
I want to make an alert based on whether the last one failed or not. If it resets to 0 we know that the redownload have fixed what last failed redownload messed up (probably).

Reply:
it can keep failing and you will see "1" all the time without knowing how many processes failed. maybe you can add a separate metric that will be incremented, next to that one?

Reply:
Yeah it can keep failing and we won't know - but for the use case I created it for it is fine as it indicates that it remains in the "probably" misaligned state. I will add another one to accumulate the failures.

Reply:
I'm just thinking right now that maybe we should add a flag if our redownload fails to disallow other nodes to redownload from us. That should prevent spreading misalignment in all of the cases.